subfun/expDesign.m

```diff
@@ -59,69 +57,68 @@
 
     % Set variables here for a dummy test of this function
```

Comment:
Will turn this into a test because this exactly how "smoke tests" work: 
feed some dummy input and make sure that the thing at least runs.